### PR TITLE
Add yaml config files for load transfer tests

### PR DESCRIPTION
**Pull request by rajkumarpatel2602**
* Added config for replication manager and replicator.

Signed-off-by: Rajkumar Patel <rajkumar.patel@seagate.com>

### CODE REVIEW

File: s3/replication/manager/tests/system/config/load_xfer_test_config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Load tranfer test configs for replication manager
```
Comment (kaustubh-d) [inline s3/replication/manager/tests/system/config/load_xfer_test_config.yaml:1]: load_transfer_test_config.yaml

File: s3/replication/manager/tests/system/load_transfer_test.py [hunk 2]
```diff
@@ -40,18 +40,29 @@
 from s3replicationcommon.templates import fdmi_record_template
 from s3replicationmanager.config import Config as ManagerConfig
 from s3_config import S3Config
+import yaml
```
Comment (kaustubh-d) [inline s3/replication/manager/tests/system/load_transfer_test.py:43]: move this above to after uuid, as a practise keep all the std imports before current package imports

File: s3/replication/manager/tests/system/load_transfer_test.py [hunk 3]
```diff
@@ -40,18 +40,29 @@
...
-    # True use random size objects, False fixed size.
-    random_size_enabled = True
-    source_bucket = "sourcebucket"
-    target_bucket = "targetbucket"
-    polling_wait_time = 5  # 5 secs wait between polling job status.
+    """Configuration for load transfer test."""
+
+    def __init__(self):
+        """Initialise."""
+
```
Comment (kaustubh-d) [inline s3/replication/manager/tests/system/load_transfer_test.py:51]: remove blank line

File: s3/replication/replicator/tests/system/config/load_xfer_test_config.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Load tranfer test configs for replicator
```
Comment (kaustubh-d) [inline s3/replication/replicator/tests/system/config/load_xfer_test_config.yaml:1]: same comment as earlier, update file name `s/xfer/transfer`

File: s3/replication/replicator/tests/system/load_transfer_test.py [hunk 5]
```diff
@@ -41,20 +41,30 @@
...
-    random_size_enabled = True
-    source_bucket = "sourcebucket"
-    target_bucket = "targetbucket"
-    polling_wait_time = 2  # 2 secs wait between polling job status.
+    """Configuration for load transfer test."""
 
-# Helper methods
+    def __init__(self):
+        """Initialise."""
+
```
Comment (kaustubh-d) [inline s3/replication/replicator/tests/system/load_transfer_test.py:54]: remove blank line

File: s3/replication/manager/tests/system/load_transfer_test.py (hunk 2 above)
Comment (rajkumarpatel2602) [reply s3/replication/manager/tests/system/load_transfer_test.py:43]: done

File: s3/replication/manager/tests/system/load_transfer_test.py (hunk 3 above)
Comment (rajkumarpatel2602) [reply s3/replication/manager/tests/system/load_transfer_test.py:51]: done

File: s3/replication/replicator/tests/system/config/load_xfer_test_config.yaml (hunk 4 above)
Comment (rajkumarpatel2602) [reply s3/replication/replicator/tests/system/config/load_xfer_test_config.yaml:1]: done

File: s3/replication/replicator/tests/system/load_transfer_test.py (hunk 5 above)
Comment (rajkumarpatel2602) [reply s3/replication/replicator/tests/system/load_transfer_test.py:54]: done

File: s3/replication/manager/tests/system/config/load_xfer_test_config.yaml (hunk 1 above)
Comment (rajkumarpatel2602) [reply s3/replication/manager/tests/system/config/load_xfer_test_config.yaml:1]: renamed files

**Review by kaustubh-d — APPROVED** [review]

**Pull request merged**